### PR TITLE
LRA-555, LRA-669  -  fix login issues

### PR DESCRIPTION
**Pull request by anantaUM**
After merging this pull request, please push a fresh copy of Gemfile.lock to Staging branch. Thank you.

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -18,7 +18,8 @@ def pundit_user
 
   def auth_user
     unless user_signed_in?
-      redirect_to root_path, notice: 'You must sign in first!'
+      $baseURL = request.fullpath
+      redirect_post(user_saml_omniauth_authorize_path, options: {authenticity_token: :auto})
     end
```
Comment (britaumich) [inline app/controllers/application_controller.rb:23]: I got the error: undefined method `redirect_post' for line 22 in the applicaton_controller

**Review by britaumich — CHANGES_REQUESTED** [review]
I got the error: undefined method `redirect_post' for line 22 in the applicaton_controller

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (anantaUM) [reply app/controllers/application_controller.rb:23]: A new gem is added. Please run delete Gemfile.lock file, run bundle install and then try again.

**Comment by anantaUM** [comment]
@britaumich - Please delete Gemfile.lock, run bundle install and then try again.

**Review by britaumich — APPROVED** [review]
Works, thanks.

**Pull request merged**